src/content/docs/new-relic-solutions/new-relic-one/core-concepts/new-relic-ai.mdx

```diff
@@ -11,12 +11,13 @@ import askgrok from 'images/grok_screenshot-crop_ask-grok.webp'
 
 import nrqlconsole from 'images/grok_screenshot-full_nrql-console.webp'
 
-New Relic AI (NRAI) is designed to help engineers improve observability. We combined OpenAI's large language models (LLMs) with our unified telemetry data platform to create an AI assistant that deeply understands how both your system and New Relic function. New Relic AI helps you demystify the complexity of your stack, solve service outages, and improve your efficiency simply by letting you ask questions in plain language. 
+import aiNewRelicAiExplainThisError from 'images/ai_screenshot-crop_new-relic-ai-explain-this-error.webp'
 
-<Video
-  type="youtube"
-  id="XMSUVmNKVyM"
-/>
+import aiExplainThisErrorPopUp from 'images/ai_screenshot-crop_explain-this-error-pop-up.webp'
+
+import aiAskAiinErrorsInbox from 'images/ai_screenshot-crop_ask-ai-in-errors-inbox.webp'
+
+New Relic AI (NRAI) is designed to help engineers improve observability. We combined OpenAI's large language models (LLMs) with our unified telemetry data platform to create an AI assistant that deeply understands how both your system and New Relic function. New Relic AI helps you demystify the complexity of your stack, solve service outages, and improve your efficiency simply by letting you ask questions in plain language. 
 
 ## Ways to interact with New Relic AI [#work-with-NRAI]
```

Comment:
I'd rewrite some of the header titles. These are just some ideas, but since the doc is long I think being able to skim the headers is really important.

* `Chat with AI` -> `Ask AI questions`
* `NRQL console` -> `Get help creating NRQL queries`
*